tests/python/common/function/test_basics.py

```diff
@@ -419,7 +417,14 @@ def _init2(shape, dtype, ctx, ids):
     # test#2: graph with no edge
     g = dgl.graph(([], []), num_nodes=5, idtype=idtype, device=F.ctx())
     g.ndata["h"] = old_repr
-    g.update_all(_message, _reduce, lambda nodes: {"h": nodes.data["h"] * 2})
+    # Intercepting the warning: The input graph for the user-defined edge
+    # function does not contain valid edges
```

Comment:
Same question as https://github.com/dmlc/dgl/pull/6146/files.

Reply:
Fixed

Reply:
As suggested in #6146, let's avoid suppressing the warning if we don't know where does it come from.

Reply:
Unlike [#6146](https://github.com/dmlc/dgl/pull/6146), here we know for sure that the graph `g`  has no edges, because that's how it was created a few lines above:
```
    g = dgl.graph(([], []), num_nodes=5, idtype=idtype, device=F.ctx())
    g.ndata["h"] = old_repr
```
I think that we can keep this warning suppression.